exercises/protein-translation/tests/proteins.rs

```diff
@@ -0,0 +1,97 @@
+extern crate protein_translation as proteins;
+
+#[test]
+fn test_methionine() {
+    let info = proteins::parse(make_pairs());
+    assert_eq!(info.name_for("ATG"), Ok("methionine"));
+}
+
+#[test]
+#[ignore]
+fn test_cysteine_tgt() {
+    let info = proteins::parse(make_pairs());
+    assert_eq!(info.name_for("TGT"), Ok("cysteine"));
+}
+
+#[test]
+#[ignore]
+fn test_cysteine_tgy() { // "compressed" name for TGT and TGC
+    let info = proteins::parse(make_pairs());
+    assert_eq!(info.name_for("TGY"), Ok("cysteine"));
+}
+
+#[test]
+#[ignore]
+fn test_stop() {
+    let info = proteins::parse(make_pairs());
+    assert_eq!(info.name_for("TAA"), Ok("stop codon"));
+}
+
+#[test]
+#[ignore]
+fn test_valine() {
+    let info = proteins::parse(make_pairs());
+    assert_eq!(info.name_for("GTN"), Ok("valine"));
+}
+
+
+#[test]
+#[ignore]
+fn test_isoleucine() {
+    let info = proteins::parse(make_pairs());
+    assert_eq!(info.name_for("ATH"), Ok("isoleucine"));
+}
+
+#[test]
+#[ignore]
+fn test_arginine_name() {
+    // In arginine CGA can be "compresed" both as CGN and as MGR
+    let info = proteins::parse(make_pairs());
+    assert_eq!(info.name_for("CGA"), Ok("arginine"));
+    assert_eq!(info.name_for("CGN"), Ok("arginine"));
+    assert_eq!(info.name_for("MGR"), Ok("arginine"));
+}
+
+#[test]
+#[ignore]
+fn test_invalid() {
+    let info = proteins::parse(make_pairs());
+    assert!(info.name_for("").is_err());
+    assert!(info.name_for("VWX").is_err()); // X is not a shorthand
```

Comment:
If we broke these out into separate tests could the test name describe what we're testing? It would remove the need for the comments.